scripts/audit-dependency-licenses.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+if test -f "decisions.yml"; then
+    decision="--decisions-file=decisions.yml"
+else
+    decision=""
```

Comment:
Why would you want to run without a decision file?  I think it the file isn't there then the tool just carries on with nothing anyway.